flexmeasures/data/scripts/cli_tasks/data_add.py

```diff
@@ -272,9 +284,20 @@ def add_beliefs(
             )
         ),
     )
-    TimedBelief.add(bdf, commit_transaction=False)
-    db.session.commit()
-    print(f"Successfully created beliefs\n{bdf}")
+    try:
+        TimedBelief.add(
+            bdf,
+            expunge_session=True,
+            allow_overwrite=allow_overwrite,
+            commit_transaction=True,
+        )
+        print(f"Successfully created beliefs\n{bdf}")
+    except IntegrityError as e:
+        db.session.rollback()
+        print(
+            f"Failed to create beliefs due to the following error: {e.orig}\n"
+            f"As a possible workaround, use the --allow-overwrite flag."
```

Comment:
I suggested we print this second line only `if allow_overwrite==False`. Otherwise, thanks for the simplification!